package-lock.json

```diff
@@ -13347,4 +13347,4 @@
       }
     }
   }
-}
+}
```

Comment:
i deleted the package-lock and then commited. so i ended up creating a new one and re-copying back the content.